pkg/discovery/servicehandler_test.go

```diff
@@ -424,7 +424,7 @@ func Test_getAuthPolicy(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			policy := getAuthPolicy(tc.policies)
+			policy, _, _ := getAuthPolicy(tc.policies)
```

Comment:
Shouldn't we assert on these other returned values?

Reply:
Lets handle this during unit test work